corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -649,6 +649,10 @@ def get_cases(case_ids, ordered=False, prefetched_indices=None):
             cases = _order_list(case_ids, cases, 'case_id')
 
         if prefetched_indices:
+            # HACK work around bug in RawQuerySetWrapper that causes
+            # next(iter(cases)) is not next(iter(cases))
```

Comment:
This is a really bad bug. Going to have to dig deep to see what other things it affects.